journal-of-zoology.csl

```diff
@@ -26,7 +26,7 @@
         <text variable="page" prefix=": "/>
         <names variable="editor translator" delimiter=", " prefix=". ">
           <name name-as-sort-order="all" and="symbol" sort-separator=", " initialize-with="." delimiter=", " delimiter-precedes-last="never"/>
-          <label form="short" prefix=" (" suffix="). " text-case="capitalize-first" strip-periods="true"/>
+          <label form="short" text-case="capitalize-first" strip-periods="true" prefix=" (" suffix=")"/>
```

Comment:
@POBrien333, the `strip-periods="true"` has to go too, right? The example shows "(Ed.).".